From 05874f93adc60f16ccd5cbeef1e2ff010cef12c1 Mon Sep 17 00:00:00 2001
From: Florian Angeletti <florian.angeletti@inria.fr>
Date: Fri, 1 Jul 2022 14:41:25 +0200
Subject: [PATCH 07/24] Merge pull request #11380 from
 damiendoligez/fix-fortran-test-on-macos

fix gfortran test on Macos

(cherry picked from commit d7a2e0fa7f2df9bd178109ba9725bb074affbd5c)
---
 testsuite/tests/lib-bigarray-2/has-gfortran.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/testsuite/tests/lib-bigarray-2/has-gfortran.sh b/testsuite/tests/lib-bigarray-2/has-gfortran.sh
index 82f7ae8aee..051122113a 100644
--- a/testsuite/tests/lib-bigarray-2/has-gfortran.sh
+++ b/testsuite/tests/lib-bigarray-2/has-gfortran.sh
@@ -5,6 +5,9 @@ if ! which gfortran > /dev/null 2>&1; then
 elif ! grep -q '^CC=gcc' ${ocamlsrcdir}/Makefile.config; then
   echo "OCaml was not compiled with gcc" > ${ocamltest_response}
   test_result=${TEST_SKIP}
+elif gcc --version 2>&1 | grep 'Apple clang version'; then
+  echo "OCaml was not compiled with gcc" > ${ocamltest_response}
+  test_result=${TEST_SKIP}
 else
   test_result=${TEST_PASS}
 fi
-- 
2.37.0.rc2

